docs/proton_c_conversion.md

```diff
@@ -8,14 +12,59 @@ You can add the same argument to your keymap's `rules.mk`, which will accomplish
 
 This exposes the `CONVERT_TO_PROTON_C` flag that you can use in your code with `#ifdef`s, like this:
 
-    #ifdef CONVERT_TO_PROTON_C
-        // Proton C code
-    #else
-        // Pro Micro code
-    #endif
+```c
+#ifdef CONVERT_TO_PROTON_C
+    // Proton C code
+#else
+    // Pro Micro code
+#endif
+```
 
-Before being able to compile, you may get some errors about `PORTB/DDRB`, etc not being defined, so you'll need to convert the keyboard's code to use the [GPIO Controls](internals_gpio_control.md) that will work for both ARM and AVR. This shouldn't affect the AVR builds at all.
+If you get errors about `PORTB/DDRB`, etc not being defined, so you'll need to convert the keyboard's code to use the [GPIO Controls](internals_gpio_control.md) that will work for both ARM and AVR. This shouldn't affect the AVR builds at all.
 
 The Proton C only has one on-board LED (C13), and by default, the TXLED (D5) is mapped to it. If you want the RXLED (B0) mapped to it instead, add this like to your `config.h`:
 
     #define CONVERT_TO_PROTON_C_RXLED
+
+## Manual Conversion
+
+To use the Proton C natively, without having to specify `CTPC=yes`, you need to change the `MCU` line in `rules.mk`:
+
+```
+MCU = STM32F303
+```
+
+Remove these variables if they exist:
+
+* `BOOTLOADER`
+* `LINK_TIME_OPTIMIZATION_ENABLE`
+* `EXTRA_FLAGS`
+
+Finally convert all pin assignments in `config.h` to the stm32 equivalents.
+
+| Pro Micro | Proton C | | Proton C | Pro Micro | Comments |
+|-----------|----------|-|----------|-----------|----------|
+| `D3` | `A9` | | 5v | RAW (5v) |
+| `D2` | `A10` | | GND | GND |
+| GND | GND | | FLASH | RESET |
+| GND | GND | | 3.3v | VCC | On a Pro Micro VCC can be 3.3v or 5v |
+| `D1` | `B7` | | `A2` | `F4` |
+| `D0` | `B6` | | `A1` | `F5` |
+| `D4` | `B5` | | `A0` | `F6` |
+| `C6` | `B4` | | `B8` | `F7` |
+| `D7` | `B3` | | `B13` | `B1` |
+| `E6` | `B2` | | `B14` | `B3` |
+| `B4` | `B1` | | `B15` | `B2` |
+| `B5` | `B0` | | `B9` | `B6` |
+| `B0` (RX LED) | `C13` | | `C13` | `D5` (TX LED) | Use `CONVERT_TO_PROTON_C_RXLED` to control the LED C13 is assigned to.
```

Comment:
`CONVERT_TO_PROTON_C_RXLED` cannot be used outside of CTPC. Not sure how is best to express the 1 led limitation within the table.

Reply:
Could have a "See note", and have a section about that.

Reply:
I broke all comments out to notes instead. Is that easier to understand?

Reply:
Clarity isnt my concern, its the fact that CTPC stuff is being mentioned in the "how to not use CTPC section", to which it still is in the table notes. CONVERT_TO_PROTON_C_RXLED is already covered above, so my suggestion is only mention that D5 is connected to the led, and RX is not connected.

Reply:
I get where you're coming from, but part of the goal here is to make manual conversion easier. I see a lot of board start out as pro-micro boards and later move to on-board arm MCU's for later revisions. I think documenting this so people know what to do when they encounter it is important.

Maybe there's a cleaner way to provide a pointer to that information?

Reply:
I revisited this today and reworded that point in a way that I think we'll both be happy with.